src/lib/errors/badRequestError.ts

```diff
@@ -0,0 +1,13 @@
+class BadRequestError extends Error {
+    private statusCode: number;
+
+    constructor(message: string) {
+        super(message);
+        this.name = "BadRequestError";
+        this.statusCode = 400;
+    }
+}
+
+export {
```

Comment:
let's just export each class directly, no need to create an intermediary object:

`export class BadRequestError ...`

Reply:
True, yes.